### DISCUSSION TITLE
Unable to include hyphens in BEM styled css modules

### DISCUSSION THREAD

**Discussion by sansyrox** [vercel/next.js#22538] [Help] [answered]
Hi Team,

My CSS classes are named as the following:

```
.app__container {
}

.app__container--visible {
}
```

I am trying to include `{styles.app__container--visible}` in my component class name, but jsx is not allowing this.

Is there a way I may be able to fix it without converting my css in a global import.

**Comment by sansyrox** [comment, marked as answer]
I just figured it out.

I used `{styles['app__container--visible']` instead

**Comment by shivaylamba** [comment]
That is great, thanks for this @sansyrox
